dataplane/standalone/saiserver/routing.go

```diff
@@ -71,6 +71,19 @@ func (n *neighbor) CreateNeighborEntry(ctx context.Context, req *saipb.CreateNei
 	return &saipb.CreateNeighborEntryResponse{}, nil
 }
 
+// CreateNeighborEntries adds multiple neighbors to the neighbor table.
+func (n *neighbor) CreateNeighborEntries(ctx context.Context, re *saipb.CreateNeighborEntriesRequest) (*saipb.CreateNeighborEntriesResponse, error) {
+	resp := &saipb.CreateNeighborEntriesResponse{}
+	for _, req := range re.GetReqs() {
+		res, err := attrmgr.InvokeAndSave(ctx, n.mgr, n.CreateNeighborEntry, req)
```

Comment:
When is Interceptor not sufficient for storing attributes in the request?

Reply:
the interceptor is only for gRPC. when invoking server funcs directly need to use InvokeAndSave